src/cc_catalog_airflow/dags/provider_api_scripts/common/licenses/constants.py

```diff
@@ -0,0 +1,138 @@
+NO_VERSION = 'N/A'
```

Comment:
Fret not:  Whenever we build a license URL using these paths, we always add the trailing slash (see `_build_license_url` in the `licenses.py` file).  I thought it better to enforce that in the one single place (the builder function) rather than making sure each of these had one individually.